### DISCUSSION TITLE
Recommended camera?

### DISCUSSION THREAD

**Discussion by jedahan** [gtxaspec/wz_mini_hacks#684] [General]
The compatibility list makes it hard for me to figure out which camera to get. V2 says its okay to latest firmware, but also won't have audio (which I would like).

The V3 says any up to 4.36.9.139, but how do we know which version firmware the camera will be ahead of time?

**Comment by bjs-pdx** [comment]
The V3 is the easiest to implement.  You can always download and flash back to 4.36.9.139 even if the camera comes with the latest and greatest.

**Reply by granthopper** [reply]
"You can always download and flash back to 4.36.9.139 "
I've been looking for the older firmware, but not having any luck.  Any tips on where to find it?
I have the Cam Pan V3. I assume I need recovery_wyzepan3_4.36.9.139.bin.zip

**Reply by gtxaspec** [reply]
Cam Pan V3 is not supported, sorry, theres no firmware to downgrade to. https://github.com/gtxaspec/wz_mini_hacks/wiki/Supported-Devices

**Reply by granthopper** [reply]
Thank you very much for letting me know (and so quickly!!!)  I did see the list and wondered (hoped) that it was simply not up to date. Actually, looking at the link you supplied, I think I was looking at some other list since I didn't even see the Cam Pan V3 listed (only V2.) I don't know if it's helpful at all, but the one I bought had a different model lumber listed on the bottom.  It was "wyzecpan3".

And thanks for putting the work in on this!  It seems like rather excellent work.  

Thanks to your information, I'll return this camera without worry that I missed something.  If I understand correctly, if I get a Wyze Cam v3, that should work if I downgrade the fw to demo_wcv3_4.36.9.139 (which I do have), correct?
